src/applications/simple-forms/21-10210/config/form.js

```diff
@@ -54,10 +55,16 @@ const pageFocus = () => {
       // since form-level useCustomScrollAndFocus is true,
       // this fn fires on every chapter change, so we need to
       // provide default focusSelector for all other pages
-      focusSelector = defaultFocusSelector;
+      focusSelector = '#nav-form-header';
     }
 
-    waitForRenderThenFocus(focusSelector);
+    if (!window.Cypress) {
```

Comment:
so much custom code for the scrolling 👀. let's keep in mind about trying to fix this at a larger scale for the future. we have a ticket in the queue to look at scrolling I think I'll be assigned to next sprint, so I may want to sync with you then.

Reply:
yeah, let's make this reusable when we have some time.   thx!